web/plates/js/status.js

```diff
@@ -68,7 +68,8 @@ function StatusCtrl($rootScope, $scope, $state, $http, $interval) {
 				var up_s = parseInt((uptime / 1000) % 60),
 					up_m = parseInt((uptime / (1000 * 60)) % 60),
 					up_h = parseInt((uptime / (1000 * 60 * 60)) % 24);
-				$scope.Uptime = String(((up_h < 10) ? "0" + up_h : up_h) + "h" + ((up_m < 10) ? "0" + up_m : up_m) + "m" + ((up_s < 10) ? "0" + up_s : up_s) + "s");
+					up_d = parseInt(uptime / (1000 * 60 * 60 * 24));
+				$scope.Uptime = String(up_d + "d" + ((up_h < 10) ? "0" + up_h : up_h) + "h" + ((up_m < 10) ? "0" + up_m : up_m) + "m" + ((up_s < 10) ? "0" + up_s : up_s) + "s");
```

Comment:
gotta love that Mod Squad.

Skypuppy
<sorry>



On 03/23/2016 10:00 AM, AvSquirrel wrote:
>
> In web/plates/js/status.js 
> <https://github.com/cyoung/stratux/pull/340#discussion_r57173858>:
>
> > @@ -68,7 +68,8 @@ function StatusCtrl($rootScope, $scope, $state, $http, $interval) {
> >  				var up_s = parseInt((uptime / 1000) % 60),
> >  					up_m = parseInt((uptime / (1000 * 60)) % 60),
> >  					up_h = parseInt((uptime / (1000 * 60 * 60)) % 24);
> > -				$scope.Uptime = String(((up_h < 10) ? "0" + up_h : up_h) + "h" + ((up_m < 10) ? "0" + up_m : up_m) + "m" + ((up_s < 10) ? "0" + up_s : up_s) + "s");
> > +					up_d = parseInt(uptime / (1000 * 60 * 60 * 24));
> > +				$scope.Uptime = String(up_d + "d" + ((up_h < 10) ? "0" + up_h : up_h) + "h" + ((up_m < 10) ? "0" + up_m : up_m) + "m" + ((up_s < 10) ? "0" + up_s : up_s) + "s");
>
> Ayup. @bkwny <https://github.com/bkwny> and I apparently implemented 
> different solutions to his request to show the day counter. I kept the 
> his formatting from d925990 
> <https://github.com/cyoung/stratux/commit/d925990455ac97c4e9ab7f98425d20635fca99df>, 
> and kept the modulo math from my branch to tighten up the code.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub 
> <https://github.com/cyoung/stratux/pull/340/files/c9ab1fd30e4700fa68a56103014d49f09b449615..c9da9eae148c2d445954d786a6d1b48674dde737#r57173858>
>